utils/parallel_runner.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"github.com/jfrog/gofrog/parallel"
+	"sync"
+)
+
+type SecurityParallelRunner struct {
+	Runner      parallel.Runner
+	ErrorsQueue chan error
+	ResultsMu   sync.Mutex
+	ScaScansWg  sync.WaitGroup // verify that the sca scan routines are done before running contextual scan
+	ScannersWg  sync.WaitGroup // verify that all scanners routines are done before cleaning temp dir
+	JasWg       sync.WaitGroup // verify that downloading analyzer manager and running all scanners are done
```

Comment:
```suggestion
    // Verify that the sca scan routines are done before running contextual scan
	ScaScansWg  sync.WaitGroup
    // Verify that all scanners routines are done before cleaning temp dir
	ScannersWg  sync.WaitGroup
    // Verify that downloading analyzer manager and running all scanners are done
	JasWg       sync.WaitGroup
```
Comment should be above and start with Capital.
* Can't we know that `ScannersWg` if both `ScaScansWg` and `JasWg` are done?